go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/aws/aws-sdk-go v1.35.3
 	github.com/basgys/goxml2json v1.1.0
 	github.com/bitly/go-simplejson v0.5.0 // indirect
+	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869
```

Comment:
Use "github.com/stretchr/testify/assert" for assertion in all tests and remove all the changes to go.mod and go.sum

Reply:
Updated